hw2/05-form.js

```diff
@@ -1 +1,27 @@
 // Add your code here
+const inputForm = document.getElementById("inputForm");
+
+const printForm = (event) => {
+  event.preventDefault();
+  const fullname = inputForm.elements[0].value;
+  const email = inputForm.elements[1].value;
+  const registrationStatus = inputForm.elements[2].value;
+  let coursesList = document.querySelector("#plCourse").checked
+    ? "Programming languages;"
+    : "";
+  if (document.querySelector("#osCourse").checked)
+    coursesList += "Operating Systems;";
+  if (document.querySelector("#fsCourse").checked)
+    coursesList += "Full Stack Web Development";
+  coursesList = coursesList.split(";").join(", ");
```

Comment:
for the course selection instead of using 'ifs', you can use a "for each" loop for this (about 3 lines of code).

Reply:
Thanks for suggesting.